Magento2/ruleset.xml

```diff
@@ -580,4 +580,13 @@
         <severity>5</severity>
         <type>warning</type>
     </rule>
+    <rule ref="Magento2.Annotation">
```

Comment:
All the rules from the following files should be added as well:
 - https://github.com/magento/magento2/blob/2.4-develop/dev/tests/static/testsuite/Magento/Test/Html/_files/html/ruleset.xml
 - https://github.com/magento/magento2/blob/2.4-develop/dev/tests/static/testsuite/Magento/Test/Less/_files/lesscs/ruleset.xml
 - https://github.com/magento/magento2/blob/2.4-develop/dev/tests/static/framework/Magento/ruleset.xml

Reply:
These ones are already there, `EndFileNewline`, `LineEndings` and `DisallowTabIndent`.
Just adding the ones missing